Lib/defcon/objects/layerSet.py

```diff
@@ -497,7 +528,7 @@ def _testLayerOrder():
     >>> layers = font.layers
     >>> layers.layerOrder
     ['public.default', 'public.background', 'Layer 1']
-    >>> layers.layerOrder = list(reversed(layers.layerOrder))
+    >>> layers.layerOrder = reversed(layers.layerOrder)
```

Comment:
`reversed` returns an iterator on both python 2 and 3. layerOrder should be a `list`.